packages/babel-traverse/src/scope/binding.js

```diff
@@ -82,6 +82,9 @@ export default class Binding {
    */
 
   reference(path: NodePath) {
+    if (this.referencePaths.indexOf(path) !== -1) {
```

Comment:
Maybe `this.referencePaths.indexOf(path) > 0` or even `this.referencePaths.includes(path)`?

(Can change reassign as well then)